DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/UserCentreDetailsDataServiceTests.cs

```diff
@@ -459,6 +459,11 @@ public void GetCentreEmailVerificationDetails_returns_expected_value()
 
             GivenEmailVerificationHashLinkedToUserCentreDetails(userId, centreId, email, code, createdDate);
 
+            connection.Execute(
+                @"UPDATE DelegateAccounts SET Approved = 1 Where UserID = @userId AND CentreID = @centreId;",
+                new { userId, centreId }
+            );

```

Comment:
Rather than doing this, it would be neater to pass a `delegateIsApproved` value into `GivenEmailVerificationHashLinkedToUserCentreDetails()`

Reply:
done

Reply:
No, I mean use the existing method rather than having the query...